tools/Vrops.py

```diff
@@ -149,368 +125,120 @@ def get_resources(self, target, token, uuids, resourcekinds):
-            logger.error(f'Problem getting property error for {propkey} - Error: {e}')
-            return False
 
-        if response.status_code == 200:
-            try:
-                if not response.json()['values']:
-                    return False
-            except json.decoder.JSONDecodeError as e:
-                logger.error(f'Catching JSONDecodeError for target {target} and property key: {propkey}'
-                             f'- Error: {e}')
-                return False
-            for resource in response.json()['values']:
-                d = dict()
-                d['resourceId'] = resource['resourceId']
-                d['propkey'] = propkey
-                content = resource['property-contents']['property-content']
-                if content:
-                    if 'values' in content[0]:
-                        d['data'] = content[0]['values'][0]
-                    else:
-                        d['data'] = content[0]['data'][0]
-                else:
-                    # resources can go away, so None is returned
-                    logger.warning(f'skipping resource for get property: {propkey}, no property-content')
-                return_list.append(d)
-
-            logger.debug(f'Amount uuids: {len(uuids)}')
-            logger.debug(f'Fetched     : {len(return_list)}')
-            logger.debug('<--------------------------------------------------')
-
-            return return_list
-        else:
-            logger.error(f'Return code: {response.status_code} != 200 for {propkey} : {response.text}')
-            return False
-
-    # if the property describes a status that has several states
-    # the expected status returns a 0, all others become 1
-    def get_latest_enum_properties_multiple(target, token, uuids, propkey, collector):
-        if not isinstance(uuids, list):
-            logger.error('Error in get project_ids: uuids must be a list with multiple entries')
-            return False
-
-        logger.debug('>-------------- get_latest_enum_properties_multiple')
-        logger.debug(f'key      : {propkey}')
-        logger.debug(f'target   : {target}')
-        logger.debug(f'collector: {collector}')
-
-        url = "https://" + target + "/suite-api/api/resources/properties/latest/query"
-        headers = {
-            'Content-Type': "application/json",
-            'Accept': "application/json",
-            'Authorization': "vRealizeOpsToken " + token
-        }
-        payload = {
-            "resourceIds": uuids,
-            "propertyKeys": [propkey]
-        }
         disable_warnings(exceptions.InsecureRequestWarning)
         try:
             response = requests.post(url,
                                      data=json.dumps(payload),
                                      verify=False,
-                                     headers=headers)
+                                     headers=headers,
+                                     timeout=30)
```

Comment:
Timeout had to be increased, otherwise vrops cannot deliver the large data structures on time.